NuKeeper.Abstractions/CollaborationModels/PullRequestRequest.cs

```diff
@@ -6,12 +6,14 @@ public PullRequestRequest(string head, string title, string baseRef)
         {
             Head = head;
             Title = title;
-            BaseRef = baseRef;
+
+            //This can be a remote that has been passed in, this happens when run locally against a targetbranch that is remote
```

Comment:
Nit: just call Replace, the check is redundant

Reply:
That's what I had before, but then every time a unnecessary replace is called.

Reply:
I think that makes sense though: https://stackoverflow.com/questions/34404042/should-i-use-string-contains-before-string-replace